.github/workflows/test.yml

```diff
@@ -32,6 +32,4 @@ jobs:
             experimental-features = nix-command flakes
             accept-flake-config = true 
       - name: Pre Commit Check
-        run: nix build .#pre-commit-check  
-      - name: Plutus Shell 
-        run: nix develop .#plutus-shell --build 
+        run: nix build .#pre-commit-check 
```

Comment:
`nix develop .#plutus-shell --build` takes too long (10 mins) and is very unlikely to break.
It will be checked by hydra anyway.

Reply:
Right, but this isn't what we did before, we checked that it _evaluated_! Is there a flake equivalent of checking whether an attribute evaluates?

Reply:
There doesn't seem to be a way to `nix flake check` a single *frament*